org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/format/LSPFormatFilesHandler.java

```diff
@@ -81,15 +82,21 @@ protected void formatFile(final @NonNull IFile file, final IProgressMonitor moni
 				return;
 
 			monitor.setTaskName(NLS.bind(Messages.LSPFormatFilesHandler_FormattingFile, file.getFullPath()));
-			final VersionedFormatRequest formatRequest = formatter.versionedRequestFormatting(doc,
-					new TextSelection(0, 0));
+			final var formatting = formatter.requestFormatting(doc,
```

Comment:
since the type is not obvious, I would prefer that we keep Optional<VersionedEdits>

Reply:
Sure